### DISCUSSION TITLE
How to disable the dropdown menu disappearing when clicked anywhere outside?

### DISCUSSION THREAD

**Discussion by nimeshmaharjan1** [saadeghi/daisyui#1853] [Q&A] [answered]
Currently the dropdown disappears when clicked anywhere other the dropdown menu how do I disable that I do not want that behaviour because I am trying to animate it with framer-motion.

**Comment by saadeghi** [comment, marked as answer]
With `dropdown-open` class you can force it to stay open (use JS to open/close it as you want):  
https://daisyui.com/components/dropdown/#force-open

Let me know if you have any questions.
